api/src/services/users.ts

```diff
@@ -161,18 +158,41 @@ export class UsersService extends ItemsService {
 	/**
 	 * Create a new user
 	 */
-	override async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
-		const result = await this.createMany([data], opts);
-		return result[0]!;
+	override async createOne(data: Partial<Item>, opts: MutationOptions = {}): Promise<PrimaryKey> {
+		try {
+			if ('email' in data) {
+				this.validateEmail(data['email']);
+				await this.checkUniqueEmails(data['email']);
```

Comment:
An error occurs during user creation as the email passed in isn't an array.

We could either wrap with an array as follows or use `toArray()` within `checkUniqueEmails()`

```suggestion
				await this.checkUniqueEmails([data['email']]);
```